src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -531,7 +551,23 @@ static ucs_status_t uct_tcp_sockcm_ep_client_invoke_connect_cb(uct_tcp_sockcm_ep
     return status;
 }
 
-ucs_status_t uct_tcp_sockcm_ep_server_handle_data_received(uct_tcp_sockcm_ep_t *cep)
+static ucs_status_t uct_tcp_sockcm_ep_client_handle_data_received(uct_tcp_sockcm_ep_t *cep)
```

Comment:
do we really need this helper function? it's used only once

Reply:
seems cleaner this way - handle function for server and handle function for the client.
there are several functions that are called only once...

Reply:
IMO the caller function is small enough to contain the code as is